tests/cannon/assert1.dora

```diff
@@ -0,0 +1,16 @@
+//= error exception
```

Comment:
You can also test the stack trace here. There are already some examples for this in the tests.

Reply:
Cool, I was looking for that, but didn't find a example. Guess I didn't look thoroghly enough.